src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Core/Errors/ValidationErrors.cs

```diff
@@ -76,4 +76,130 @@ private static readonly ValidationErrorDescriptorFactory _factory
     /// </summary>
     public static ValidationErrorDescriptor AssignmentIsActiveInOneOrMoreDelegations { get; }
         = _factory.Create(10, $"Assignment is active in one or more delegations.");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor NotAuthorizedForConsentRequest { get; }
+        = _factory.Create(19, "Not Authorized for ");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor InvalidPersonIdentifier { get; }
+        = _factory.Create(20, "Invalid person identifier. No person with this identifier.");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor InvalidOrganizationIdentifier { get; }
+        = _factory.Create(21, $"Invalid organization identifier. No organization with this identifier.");
+
+    /// <summary>
+    /// Gets a validation error descriptor
+    /// </summary>
+    public static ValidationErrorDescriptor InvalidValidToTime { get; }
```

Comment:
Validation errors should not really be field specific, just type specific. Instead of calling this `InvalidValidToTime`, it should be `TimeNotInFuture`, and the message should read something like `Time needs to be in the future`. The field-name is provided where validaiton-errors are used, and as such a validation error can be used for fields withdifferent names.